apps/drec-api/src/pods/device-group/dto/new-device-group.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsString, IsArray } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class NewDeviceGroupDTO {
+  @ApiProperty()
+  @IsString()
+  name: string;
+
+  @ApiProperty()
+  @IsArray()
```

Comment:
Please validate whatever is inside array. Class validator has `each: true` method on normal validators, that make them validate arrays

Reply:
Yeah but I don`t see it working with @IsNumber({each:true}). It works with  @IsString({each: true}) but not with IsNumber.
And to add this breaks the code:
```
  @IsArray()
  @ValidateNested({ each: true })
  @Type(() => Number)
```

Reply:
@Malak67 you can do

```js
@IsInt({ each: true })
@Min(1, { each: true })
```

This one works for sure, we use it in Origin.